pallets/loans-ref/src/benchmarking.rs

```diff
@@ -221,14 +224,21 @@ where
 	fn initialize_active_state(n: u32) -> PoolIdOf<T> {
 		let pool_id = Self::prepare_benchmark();
 
+		// TODO: ideally should this be computed by a parameter?
 		for i in 1..MaxRateCountOf::<T>::get() {
 			// First `i` (i=0) used by the loan's interest rate.
 			let rate = T::Rate::saturating_from_rational(i + 1, 5000);
 			T::InterestAccrual::reference_rate(rate).unwrap();
 		}
 
+		// TODO: ideally should this be computed by a parameter?
 		for i in 0..MaxCollectionSizeOf::<T>::get() {
-			let price_id = T::PriceId::from(i + 1);
+			let price_id = i.into();
+			// This account is different in each iteration because of how oracles works.
+			// This restriction no longer exists once
+			// https://github.com/open-web3-stack/open-runtime-module-library/pull/920 is merged
+			let feeder = account("feeder", i, 0);
+			T::PriceRegistry::feed_value(feeder, price_id, 0.into()).unwrap();
```

Comment:
This is quite tricky. Because of how oracles are implemented, the same user can not feed twice in the same block. And the internal trait they use (`DataFeeder`) only allows us to feed one price id each time.

That's why we need to create a new account per the price we want price here. And consequently, we need to mock the member's pallet in the runtime. To allow to use all these users.